pkg/storkctl/migrationschedule.go

```diff
@@ -197,6 +199,16 @@ func newCreateMigrationScheduleCommand(cmdFactory Factory, ioStreams genericclio
 				includeOptionalResourceTypes = append(includeOptionalResourceTypes, "Job")
 			}
 
+			//validate excludeResourceTypes
+			if len(excludeResourceTypes) != 0 {
+				resourceTypes := strings.Join(excludeResourceTypes, ",")
+				excludeResourceTypes, err = getResourceTypes(resourceTypes, ioStreams)
```

Comment:
I am reusing getResourceTypes function from applicationBackup code path.
Discussed with Dipti that this cannot be unit tested because of the aeclient.Discovery() being used in the code